references.bib

```diff
@@ -1,1167 +1,1164 @@
+	Biburl = {https://dblp.org/rec/bib/conf/sp/Das0MK18},
+	Booktitle = {2018 {IEEE} Symposium on Security and Privacy, {SP} 2018, Proceedings, 21-23 May 2018, San Francisco, California, {USA}},
+	Crossref = {DBLP:conf/sp/2018},
+	Date-Added = {2019-01-29 02:25:00 +0100},
+	Date-Modified = {2019-01-29 02:25:00 +0100},
+	Doi = {10.1109/SP.2018.00011},
+	Pages = {108--126},
+	Timestamp = {Fri, 24 Aug 2018 00:34:59 +0200},
+	Title = {Anonymity Trilemma: Strong Anonymity, Low Bandwidth Overhead, Low Latency - Choose Two},
+	Url = {https://doi.org/10.1109/SP.2018.00011},
+	Year = {2018},
+	Bdsk-Url-1 = {https://doi.org/10.1109/SP.2018.00011}}
+
+@proceedings{DBLP:conf/sp/2018,
+	Bibsource = {dblp computer science bibliography, https://dblp.org},
+	Biburl = {https://dblp.org/rec/bib/conf/sp/2018},
+	Date-Added = {2019-01-29 02:25:00 +0100},
+	Date-Modified = {2019-01-29 02:25:00 +0100},
+	Publisher = {{IEEE}},
+	Timestamp = {Thu, 09 Aug 2018 08:55:11 +0200},
+	Title = {2018 {IEEE} Symposium on Security and Privacy, {SP} 2018, Proceedings, 21-23 May 2018, San Francisco, California, {USA}},
+	Url = {http://ieeexplore.ieee.org/xpl/mostRecentIssue.jsp?punumber=8418581},
+	Year = {2018},
+	Bdsk-Url-1 = {http://ieeexplore.ieee.org/xpl/mostRecentIssue.jsp?punumber=8418581}}
+
+@article{DBLP:journals/csur/ShiraziSABD18,
+	Author = {Fatemeh Shirazi and Milivoj Simeonovski and Muhammad Rizwan Asghar and Michael Backes and Claudia D{\'{\i}}az},
+	Bibsource = {dblp computer science bibliography, https://dblp.org},
+	Biburl = {https://dblp.org/rec/bib/journals/csur/ShiraziSABD18},
+	Date-Added = {2019-01-29 01:58:26 +0100},
+	Date-Modified = {2019-01-29 01:58:26 +0100},
+	Doi = {10.1145/3182658},
+	Journal = {{ACM} Comput. Surv.},
+	Number = {3},
+	Pages = {51:1--51:39},
+	Timestamp = {Wed, 21 Nov 2018 12:44:28 +0100},
+	Title = {A Survey on Routing in Anonymous Communication Protocols},
+	Url = {https://doi.org/10.1145/3182658},
+	Volume = {51},
+	Year = {2018},
+	Bdsk-Url-1 = {https://doi.org/10.1145/3182658}}
+
+@inproceedings{DBLP:conf/uss/AlexopoulosKT017,
+	Author = {Nikolaos Alexopoulos and Aggelos Kiayias and Riivo Talviste and Thomas Zacharias},
+	Bibsource = {dblp computer science bibliography, https://dblp.org},
+	Biburl = {https://dblp.org/rec/bib/conf/uss/AlexopoulosKT017},
+	Booktitle = {26th {USENIX} Security Symposium, {USENIX} Security 2017, Vancouver, BC, Canada, August 16-18, 2017.},
+	Crossref = {DBLP:conf/uss/2017},
+	Date-Added = {2019-01-29 01:58:04 +0100},
+	Date-Modified = {2019-01-29 01:58:04 +0100},
+	Pages = {1217--1234},
+	Timestamp = {Fri, 01 Sep 2017 12:29:11 +0200},
+	Title = {MCMix: Anonymous Messaging via Secure Multiparty Computation},
+	Url = {https://www.usenix.org/conference/usenixsecurity17/technical-sessions/presentation/alexopoulos},
+	Year = {2017},
+	Bdsk-Url-1 = {https://www.usenix.org/conference/usenixsecurity17/technical-sessions/presentation/alexopoulos}}
+
+@proceedings{DBLP:conf/uss/2017,
```

Comment:
can you remove the conference entry, as it is less useful that the more specific pointer you provide above the the individual paper within it.